### DISCUSSION TITLE
Can you create a method on the dao, with an abstract class?

### DISCUSSION THREAD

**Discussion by HasslerYahir** [vitusortner/floor#575] [General]
Can you create a method on the dao, with an abstract class?
for example 
  @insert
  Future<List<int>> inserBeneficiary(List<Beneficiary> beneficiary);

being an abstract class beneficiary.

**Comment by peace-shillong** [comment]
Yes it's possible. 
I wanted to create an insert method in the dao that allows me to insert an entire List into the database without the need of inserting each item inside a for loop.

`
  @Insert(onConflict: OnConflictStrategy.replace)
  Future<List<int>> insertBeneficiary(List<Beneficiary> Beneficiary);
`
where 
Beneficiary has a primary key id of int type.
